Add new watch configuration doc and rename "Actions" (#1104)
home-assistant/companion.home-assistant: docs/apple-watch/apple-watch.md

id: "apple-watch"
---

> Home Assistant has deep integration with the Apple Watch. You can display Home Assistant information as complications directly on your watch face and since app version 2024.9, run scripts, active scenes and iOS Actions.

Reviewer: ```suggestion
Home Assistant has deep integration with the Apple Watch. You can display Home Assistant information as complications directly on your watch face and since app version 2024.9, run scripts, active scenes, and iOS Actions.
```

## Home

> Since iOS App version 2024.9 release, you can now customize your Home Assistant watch experience using `Scripts`, `Scenes` and `iOS Actions`.

Reviewer: ```suggestion
Since iOS App version 2024.9 release, you can now customize your Home Assistant watch experience using `Scripts`, `Scenes`, and `iOS Actions`.
```

Author: No need for `,` when there is an 'and' afterwards right?

Reviewer: it depends... :-D we do the Oxford comma. Meaning, if you give a list of things, a comma is used even in front of the and. It improves readability
If and is used to connect two separate clauses, then no comma is used.

> Using your iPhone, open companion app settings and look under 'Apple Watch' section for 'Configuration', if you already had `iOS Actions` created, they will be migrated into the configuration screen, just hit 'Save' and reload on your watch.

Reviewer: ```suggestion
Using your iPhone, open the Companion app settings and under the **Apple Watch** section, look for **Configuration**. If you already had **iOS Actions** created, they will be migrated into the configuration screen. Select **Save** and reload on your watch.
```
we usually use bold to markup UI strings
